website/blog/2023-09-26-ferretdb-releases-v1110.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: ferretdb-releases-v1110
+title: FerretDB releases v1.11.0
+authors: [alex]
+description: >
+  We just released FerretDB v1.11.0, which includes several improvements for the SQLite backend.
+image: /img/blog/ferretdb-v1.11.jpg
+tags: [release]
+---
+
+![FerretDB v.1.11.0 - new release](/img/blog/ferretdb-v1.11.jpg)
+
+We just released FerretDB v1.11.0, which includes several improvements for the SQLite backend.
+
```

Comment:
Can we mention our work on the PostgreSQL backend earlier in the article? Just to avoid the look that we abandoned Postgres, since that is very farm from the truth.